metriq-api/service/platformDataTypeValueService.js

```diff
@@ -0,0 +1,29 @@
+// platformDataTypeValueService.js
+
+// Data Access Layer
+const ModelService = require('./modelService')
+// Model
+const db = require('../models/index')
+const PlatformDataTypeValue = db.platformDataTypeValue
+
+class PlatformDataTypeValueService extends ModelService {
+  constructor () {
+    super(PlatformDataTypeValue)
+  }
+
+  async validate (propertyTypeValue) {
+    if (propertyTypeValue.value === undefined) {
+      return { success: false, error: 'Property value cannot be blank.' }
```

Comment:
'Property value cannot be blank.' -> 'Property platform value cannot be blank.'

Reply:
Ideally, it's easiest or best to have one "model," per one "controller," per one "view." However, "properties" or "platform properties" are "domain logic" that expose a "logical model" data transfer object (DTO) on 2 or more database table models. I see your point, but I think this actually the less confusing message option for the end user, who only sees a "property DTO."